addons/mail/tests/test_mail_render.py

```diff
@@ -211,6 +212,23 @@ def test_render_template_inline_template(self):
             )[partner.id]
             self.assertEqual(rendered, expected)
 
+    @users('employee')
+    def test_render_template_inline_template_w_post_process_custom_local_links(self):
+        def _mock_get_base_url(recordset):
+            return f"http://www.render-object-{recordset.id}.com"
+        target_records = (self.render_object + self.render_object_fr).with_user(self.env.user)
+        with patch('odoo.models.Model.get_base_url', new=_mock_get_base_url):
+            render_results = self.env['mail.render.mixin']._render_template(
+                '<a href="/test/destination"><img src="/test/image"></a>',
+                target_records._name,
+                target_records.ids,
+                engine='inline_template',
+                post_process=True,
+            )
+        for partner_id, render_result in render_results.items():
```

Comment:
Could be interesting to have some query counter checks, and be sure the prefetch effectively works

Reply:
Indeed, might get expensive if it doesn't ^^

Added a query count check above, and a test commit to add the test before fixing. For documentation purposes.

Reply:
No query counters before / after, to see the difference ?

Reply:
True

1 (fetch parameter) -> 6 (check access rights, rules and companies + fetch name)